src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStepExecution.java

```diff
@@ -184,6 +189,9 @@ public HttpResponse proceed(@CheckForNull Map<String,Object> params) {
         }
         node.addAction(new InputSubmittedAction(approverId, params));
 
+        // notify this input action will proceed
+        notifyInput(InputExtension.NotifyEvent.PROCEED);
```

Comment:
Execute before return ? in case some exceptions

Reply:
Same with above.